src/main/java/com/gmail/yuramitryahin/service/UserService.java

```diff
@@ -0,0 +1,15 @@
+package com.gmail.yuramitryahin.service;
+
+import com.gmail.yuramitryahin.entity.User;
+
+public interface UserService {
+    void add(User user);
+
+    void update(User user);
+
+    User getById(Long id);
+
+    User getByPhoneNumber(String phoneNumber);
+
+    void delete(User user);
```

Comment:
maybe it is better to have deleteById() method?

Reply:
yep